### DISCUSSION TITLE
GUI coloring

### DISCUSSION THREAD

**Discussion by MechManDan** [supermerill/SuperSlicer#1860] [Q&A]
Hello Maybe I am missing something but is there a way to change the color of the GUI to more of a dark theme i am not a fan of bright background with dark text and I would love to have a way to have a black main interface with white or colored text thanks

**Comment by supermerill** [comment]
All the standard gui colors are given by your os. I don't have any control on them.
There is the accent color available in the gui preferences.
You can modify a bit more colors (gcode preview color and default plater color) in the ui_layout/colors.ini

**Reply by MechManDan** [reply]
Darn I wonder if Slicer is not picking up the windows Dark mode Setting, I would like to change the color of the areas circled and get them to be a black background with white text if possible.... 

would this be something I can customize in the colors.ini you mentioned?


![Screenshot 2021-11-12 163927](https://user-images.githubusercontent.com/5958359/141595758-4e9d4448-4895-4cc7-8194-af1f390c1ee7.png)

**Reply by supermerill** [reply]
no.
I think prusa has done something like that for the next release, so it may comes.
Note that "changing some colors" isn't enough. It's reskinning the entire GUI & gui libraries and it's a very big task, many man-month of work.

**Comment by neophyl** [comment]
Windows doesn't have a proper dark mode api.  Which is why its been giving the Prusa devs fits.  They have basically had to look at all the workarounds other windows software like Notepadd++ has been using on Windows and try and adapt them to PS.

I still dont think they have been 100% successful as it seems some gui combinations are still causing issues.
